src/plugins/home/server/services/sample_data/routes/install.ts

```diff
@@ -135,14 +148,13 @@ export function createInstallRoute(
           const createIndexParams = {
             index,
             body: {
-              settings: { index: { number_of_shards: 1, auto_expand_replicas: '0-1' } },
+              settings: dataSourceId
+                ? { index: { number_of_shards: 1 } }
```

Comment:
we can still have auto expand replica to `0-1`

Reply:
During local test, I notice this number varies base on how many data nodes my data source have. -  e.g. If single node, the number needs to be 0-3. Otherwise, it will error out with: "expected max cap on auto expand to be a multiple of total awareness attributes [3]" from: https://github.com/opensearch-project/OpenSearch/blob/743eed4c8ecb11c4898d340c0127c84347df5b40/server/src/main/java/org/opensearch/cluster/routing/allocation/AwarenessReplicaBalance.java#L109

Reply:
I am ok to set to no replica for sample data then.